
public class D22Throw_Statement {

	public static void main(String[] args) {
		// TODO Auto-generated method stub
		try {
			compute(9);
			compute(100);
		}
		catch(Exception e) {
			System.out.println(e);
		}
	}
	
	public static void compute(int i ) 
	throws Exception{
		if (i < 10)
			System.out.println("Normal Execution"); 
		else 
			throw new Exception("erroe due to "+i);
		
	}
	
}
